app/components/__snapshots__/Body.test.tsx.snap

```diff
@@ -15,79 +15,74 @@ exports[`Body > Body snapshot admin 1`] = `
         class="drawer-content flex flex-col"
       >
         <nav
-          class="navbar absolute left-0 top-0 z-10 h-16 bg-primary text-primary-content shadow-md"
+          class="navbar absolute left-0 top-0 z-10 h-16 justify-between bg-primary text-primary-content shadow-md"
         >
           <div
-            class="flex-none lg:hidden"
+            class="flex"
           >
-            <label
-              class="btn btn-square btn-ghost"
-              for="drawer"
+            <div
+              class="flex-none lg:hidden"
             >
-              <svg
-                class="inline-block h-6 w-6 stroke-current"
-                color="currentColor"
-                fill="none"
-                height="1.5em"
-                stroke-width="1.5"
-                viewBox="0 0 24 24"
-                width="1.5em"
-                xmlns="http://www.w3.org/2000/svg"
+              <label
+                class="btn btn-square btn-ghost"
+                for="drawer"
               >
-                <path
-                  d="M3 5H21"
-                  stroke="currentColor"
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                />
-                <path
-                  d="M3 12H21"
-                  stroke="currentColor"
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                />
-                <path
-                  d="M3 19H21"
-                  stroke="currentColor"
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                />
-              </svg>
-            </label>
-          </div>
-          <div
-            class="flex-1"
-          >
+                <svg
```

Comment:
@kbario given that we are here what about using the icons from here https://iconoir.com/, instead of SVGs?

Reply:
Hey Luca, this is generated code for the test, the component it tests uses the icon noir icon